packages/cli/package.json

```diff
@@ -17,24 +17,47 @@
         "lib/*"
     ],
     "scripts": {
-        "build": "$npm_execpath run clean && tsc",
+        "build": "$npm_execpath run clean && tsc -p tsconfig.build.json",
         "clean": "rimraf lib",
-        "dev": "tsc --watch",
+        "dev": "tsc --watch --preserveWatchOutput -p tsconfig.build.json",
         "lint": "run-p lint:prettier lint:eslint lint:tsc",
         "lint:eslint": "eslint --max-warnings 0 src/ **/*.json --no-warn-ignored",
         "lint:prettier": "npx prettier --check './**/*.{js,json,md,yml,yaml}'",
-        "lint:tsc": "tsc"
+        "lint:tsc": "tsc",
+        "test": "NODE_OPTIONS=--experimental-vm-modules npx jest",
+        "test:watch": "jest --watch"
     },
     "dependencies": {
+        "@comet/cms-api": "workspace:*",
```

Comment:
Why does this need this dependency? The CLI package is used in Admin and site, which would then need to install all API peer dependencies.